src/midcom/datamanager/extension/type/select.php

```diff
@@ -44,9 +44,18 @@ public function configureOptions(OptionsResolver $resolver)
             }
         };
 
+        $multiple_options = function (Options $options) {
+            $return_options = array();
+            if (isset($options['type_config']['allow_multiple'])) {
+                $return_options = $options['type_config']['allow_multiple'];
+            }
+            return $return_options;
+        };
```

Comment:
P.S.: for consistency, it would be better to call the function `$map_multiple` or something similar